content/en/docs/overview/variables.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Variables"
+weight: 210
+---
+
```

Comment:
I think there should be a short introduction to variables here, even if the target audience is developers.

Reply:
Added a sentence to explain that variables are statically typed.

@jyniybinc  Would you explain this differently? What else would you put into the introduction?